js/map.js

```diff
@@ -1,22 +1,46 @@
 var map = null;
 var infowindow = null;
 var nodes = [];
+var markers = [];
 var links = [];
 var currentNodeListURL;
 var circle = null;
 var mapStyle;
 var urlBase = '';
 
+// Process URL params
+function getURLParam(key) {
+  return new URLSearchParams(window.location.search).get(key);
+}
+
 function initialize() {
   //Current Node URL with random bits to make sure it doesnt get cached
-  currentNodeListURL = document.getElementById('nodeURL').value + '?ramd=' + new Date();
-  urlBase=currentNodeListURL.substring(0,currentNodeListURL.lastIndexOf('/'));
+  currentNodeListURL = document.getElementById('nodeURL').value;
+  
 
   //Set options based on check box positions
   var filterActive = document.getElementById('chkActive').checked;
   var filterProposed = document.getElementById('chkProposed').checked;
+  var filterBuildings = document.getElementById('chkTheme').checked;  
```

Comment:
Since the pattern `document.getElementById('chkTheme').checked` is repeated so many times it could be abstracted into a function. Then it would be easier to guard against checking for the `checked` property of `null`